### PR TITLE
Sky130 various updates

### PR DESCRIPTION
**Pull request by msaligane**
@jjcherry56 @rovinski added the sdc fixes + gds files

### CODE REVIEW

**Comment by tajayi** [comment]
Any chance we can merge the gds files into a single one? (Like Nangate)

**Comment by msaligane** [comment]
@tajayi we could. It will be done soon.

File: flow/util/markDontUse.py [hunk 1]
```diff
@@ -25,7 +25,10 @@
 f.close()
 
 # Pattern to match a cell header
-pattern = r"(^\s*cell\s*\(\s*("+"|".join(patternList)+")\)\s*\{)"
+#pattern = r"(^\s*cell\s*\(\s*("+"|".join(patternList)+")\)\s*\{)"
+#pattern = r"(^\s*cell\s*\([\"]\s*("+"|".join(patternList)+")\[\"])\s*\{)" 
+pattern = r"(^\s*cell\s*\(\s*([\"]*"+"[\"]*|[\"]*".join(patternList)+"[\"]*)\)\s*\{)"
+
```
Comment (rovinski) [inline flow/util/markDontUse.py:31]: Please remove commented code.

Did you verify this still works with other libraries?

Comment (msaligane) [reply flow/util/markDontUse.py:31]: Yes, I will double check.

Comment (msaligane) [reply flow/util/markDontUse.py:31]: This works for other technology nodes as well

File: flow/Makefile [hunk 2]
```diff
@@ -46,11 +46,11 @@
 # DESIGN_CONFIG=./designs/gf14/bp_dual/config.mk
 # DESIGN_CONFIG=./designs/gf14/bp_quad/config.mk
 
-# DESIGN_CONFIG=./designs/sky130/gcd/config.mk
+ DESIGN_CONFIG=./designs/sky130/gcd/config.mk
 
 
 # Default design
-DESIGN_CONFIG ?= ./designs/nangate45/gcd/config.mk
+#DESIGN_CONFIG ?= ./designs/nangate45/gcd/config.mk
```
Comment (tajayi) [inline flow/Makefile:53]: I think we should leave the default as nangate45/gcd for now

Comment (msaligane) [reply flow/Makefile]: Done

**Comment by msaligane** [comment]
@tajayi @rovinski no merged LEF yet. But will add that as soon as I have one ready. Thanks.
Please review again and merge if okay.

**Comment by rovinski** [comment]
@msaligane merged LEF or merged GDS? I'm still waiting on the single GDS. Everything else looks fine.

**Comment by msaligane** [comment]
Sorry I meant no merged gds yet...

**Comment by msaligane** [comment]
@Rovinski Again there are no merged gds yet. This works with separate gds. I will add a merged gds when I have it. I see no reason to stall this. Please merge as it is fixing a few issues as well.

**Comment by rovinski** [comment]
@msaligane A merged GDS can be created with any GDS tool (e.g. klayout). I was going to merge this change via squash so that the extra overhead from all the GDSs would be deleted from the repo.

 The other option is to remove the GDSs and make a separate PR with the single GDS.

**Comment by msaligane** [comment]
@rovinski Okay, I will try to merge in klayout. But do you know if the gds size going to be different if they are separate or a single file? I know a single large file is usually the issue.

**Comment by rovinski** [comment]
The file size is generally not that different. The problem is that git sees binary files mostly as blackboxes. So every change to a binary file creates an entirely new copy in the git history. So we want to commit the GDS once and then ideally not touch it after that. Maybe we need to set up git LFS but that's seems like yet another time sink right now.

**Comment by msaligane** [comment]
Okay, sounds good. I will try to merge it today.

**Comment by tajayi** [comment]
Yeah, github has a 100MB limit on any individually tracked file. This shouldn't be a problem though. The largest gds file is about 26KB. Even if we multiply that by 400 it's only about 10MB

File: flow/Makefile (hunk 2 above)
Comment (rovinski) [reply flow/Makefile:53]: Please un-redo this change. We do not want to change the default.

**Pull request merged**